docs/source/commandline/commandline-all.rst

```diff
@@ -1592,32 +2213,45 @@ Core Communication Options
 
 **-comm_localhost** *<string>*
 
-	Local address to bind to. This can be a traditional ``xxx.xxx.xxx.xxx`` address or a string containing a resolvable hostname. The default is value is "``0.0.0.0``"
+    Local address to bind to. This can be a traditional ``xxx.xxx.xxx.xxx``
+    address or a string containing a resolvable hostname.
+
+    The default is value is "``0.0.0.0``"
```

Comment:
Add an explanatory note "... ``0.0.0.0`` (bind to all local IPv4 addresses)."

Reply:
Done.